### PR TITLE
Google Scholar author

### LINKED ISSUE #176: Author Results

### PR DESCRIPTION
**Pull request by Mykhailo-Zub**
Resolves #176. [Draft](https://serpapi.com/blog/p/9c7e783b-aff7-4a32-b656-b38f22cdc5f5/)

### CODE REVIEW

File: Google/google_scholar/author_profile_results/javascript/author_results/google-scholar-author-results-scrape.md [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+    website,
+    interests:
+      interests?.map((interest) => {
+        const { title, link = "link not available" } = interest;
+        return {
+          title,
+          link,
+        };
+      }) || "no interests",
+    articles: articles?.map((article) => {
```
Comment (dimitryzub) [inline Google/google_scholar/author_profile_results/javascript/author_results/google-scholar-author-results-scrape.md:401]: Let's also scrape all author's articles in this blog post.

File: Google/google_scholar/author_profile_results/javascript/author_results/google-scholar-author-results-scrape.md [hunk 2]
```diff
@@ -0,0 +1,675 @@
...
+      photo: buildValidLink($("#gsc_prf_pup-img").attr("src")),
+      affiliations: $(".gsc_prf_il:nth-child(2)").text().trim(),
+      website: $(".gsc_prf_ila").attr("href") || "website not available",
+      interests: Array.from($("#gsc_prf_int a")).map((interest) => {
+        return {
+          title: $(interest).text().trim(),
+          link: buildValidLink($(interest).attr("href")),
+        };
+      }),
+      articles: Array.from($(".gsc_a_tr")).map((el) => {
```
Comment (dimitryzub) [inline Google/google_scholar/author_profile_results/javascript/author_results/google-scholar-author-results-scrape.md:61]: Let's also scrape all author's articles in this blog post.

**Review by dimitryzub — CHANGES_REQUESTED** [review]
@Mykhailo-Zub Thank you so much! ⚡

I've added a few suggestions and for big suggestion changes, let's add all author articles extractions in this blog post also.

- author info
- all articles instead of articles from the first page
- graph
- co-authors

**Review by dimitryzub — CHANGES_REQUESTED** [review]
@Mykhailo-Zub Thank you 🥇

One final thing, could you please update replit example for `puppeteer`? 

image

**Comment by dimitryzub** [comment]
@Mykhailo-Zub Thank you for the fix 👍 You can go ahead and merge this PR. Labeling this blog as accepted. Really good job, thank you.

Let's @set-reminder July 1 5am @Mykhailo-Zub publish #243 blog post if can.

**Comment by Mykhailo-Zub** [comment]
https://serpapi.com/blog/how-to-scrape-google-author-with-node-js/
https://medium.com/@mikhail.a.zub/how-to-scrape-google-scholar-author-info-with-node-js-2c4114c73af9
https://dev.to/serpapi/how-to-scrape-google-scholar-author-info-with-nodejs-1b37
https://serpapi.hashnode.dev/how-to-scrape-google-scholar-author-info-with-nodejs

**Pull request merged**